docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -346,7 +348,7 @@ services:
     restart: always
   vmanomaly:
     container_name: vmanomaly
-    image: us-docker.pkg.dev/victoriametrics-test/public/vmanomaly-trial:v1.5.0
+    image: us-docker.pkg.dev/victoriametrics-test/public/vmanomaly-trial:v1.7.0
```

Comment:
putting here just not to forget - will be substituted to `v1.7.2` after we merge the hotfix and release a new one